### DISCUSSION TITLE
change installation path to programFiles and not to programFiles(x86) not possible

### DISCUSSION THREAD

**Discussion by karimmohammad** [oleg-shilo/wixsharp#1425] [Q&A]
hello,

first a very nice new year and health to you.

my question:
everytime when I try to change the installation path to a directory inside programFiles the installer installs the files into programFiles(x86). 
for example I want to install into the directory programFiles/test the installer creates a directory test into the programFiles(x86) directory. do you think I made a mistake or it is not possible to install into programFiles directory?

kind regards
Karim

**Comment by oleg-shilo** [comment]
WiX setup is a x86 application by default. It's not WixSharp but WiX design limitation.
Thus if you want to target x64 Win then you need to indicate this explicitly. Fortunately, with WixSharp it is just a single property assignment.
See this example: https://github.com/oleg-shilo/wixsharp/blob/master/Source/src/WixSharp.Samples/Wix%23%20Samples/Install%20on%20x64/setup.cs
